thunder/executors/passes.py

```diff
@@ -129,8 +129,16 @@ def visit_(bsym: BoundSymbol) -> transforms.VISIT_TYPE:
 
 
 def transform_for_execution(trace: TraceCtx, executors_list: Sequence[Executor]) -> TraceCtx:
+    import torch
+
     start_time_ns = time.time_ns()
 
+    if torch.distributed.is_available():
+        # Apply AllReduce bucketing if possible & needed
+        from thunder.distributed.transforms.ddp import apply_bucketing_to_grad_allreduce
```

Comment:
Is `apply_bucketing_to_grad_allreduce` importable with PyTorch build without distributed available? If yes could this if available statement be moved inside the function to keep the indentation in `transform_for_execution` cleaner?